src/mds/MDSMap.h

```diff
@@ -626,6 +631,11 @@ class MDSMap {
   mds_rank_t standby_count_wanted = -1;
   std::string balancer;    /* The name/version of the mantle balancer (i.e. the rados obj name) */
 
+  std::string bal_rank_mask;
+  std::string bal_rank_mask_bin_string;
```

Comment:
We only need to create the intermediate bit string when generating the bitset. No need to keep in the MDSMap?

Reply:
bal_rank_mask_bin_string is removed.